docs/token_authorization_api.md

```diff
@@ -1,11 +1,14 @@
 # TorchServe token authorization API
 
-## Setup
-1. Download the jar files from [Maven](https://mvnrepository.com/artifact/org.pytorch/torchserve-endpoint-plugin) 
-2. Enable token authorization by adding the `--plugins-path /path/to/the/jar/files` flag at start up with the path leading to the downloaded jar files.
+Torchserve now supports token authorization by default.
+
+## How to disable Token Authorization
+1. Set global environment variable `TS_DISABLE_TOKEN_AUTHORIZATION=true`
+2. Add `--disable-token` to command line when running TorchServe.
+3. Add `disable_token_authorization=true` to config.properties file
```

Comment:
Should we clarify that any one of these options can be used to disable token authorization?
```suggestion
## How to disable Token Authorization
1. Set global environment variable `TS_DISABLE_TOKEN_AUTHORIZATION=true` or
2. Add `--disable-token` to command line when running TorchServe or
3. Add `disable_token_authorization=true` to config.properties file
```

Also for `Set global environment variable TS_DISABLE_TOKEN_AUTHORIZATION=true` I think it may be good to link to https://github.com/pytorch/serve/blob/master/docs/configuration.md and mention that `enable_envvars_config=true` must be set to ensure that the environment variable takes effect.